documentation/en/architecture/mpool.md

```diff
@@ -52,7 +52,7 @@ Use `MpoolPushMessage` instead.
 ### MpoolSub
 
 Returns a channel to receive notifications about updates to the message pool.
-Note that the context *must* be cancelled when the caller is done with the subscription.
+Note that the context *must* be canceled when the caller is done with the subscription.
```

Comment:
```suggestion
Note that the context *must* be cancelled when the caller is done with the subscription.
```

this was fine as it was, it's the preferred spelling outside of the US but also valid in the US